book/src/keymap.md

```diff
@@ -270,6 +270,8 @@ Mappings in the style of [vim-unimpaired](https://github.com/tpope/vim-unimpaire
 | `[c`     | Go to previous class (**TS**)                | `goto_prev_class`     |
 | `]p`     | Go to next parameter (**TS**)                | `goto_next_parameter` |
 | `[p`     | Go to previous parameter (**TS**)            | `goto_prev_parameter` |
+| `]o`     | Go to next comment (**TS**)                  | `goto_next_comment`   |
+| `[o`     | Go to previous comment (**TS**)              | `goto_prev_comment`   |
```

Comment:
This doesn't seem intuitive for me, I think `]#` might be better.

Reply:
Only problem with # is that it is a shifted key usually, making it harder to quickly use, and breaks the pattern of using a letter for the other textobjects. That said, maybe we open an issue for further discussion?